packages/discord.js/src/structures/Presence.js

```diff
@@ -216,6 +216,13 @@ class Activity {
      */
     this.party = data.party ?? null;
 
+    /**
+     * The sync id of the activity
+     * <info>This represents the track id in spotify activities.</info>
```

Comment:
Not sure if we should mention this or let users figure it out

Reply:
that is the prime (if not only) use case for it at the moment, seems worth mentioning IMO for ease of use